minutus-mruby-build-utils/src/lib.rs

```diff
@@ -86,23 +122,44 @@ impl MRubyManager {
 }
 
 fn build_mruby(workdir: &Path, path: &Path) {
+    let rake = if cfg!(windows) { "rake.bat" } else { "rake" };
+
     let c = &[
-        "rake",
+        rake,
         "all",
         &format!("MRUBY_CONFIG={}", path.to_string_lossy()),
     ];
     run_command(&workdir.join("mruby"), c).unwrap();
 }
 
 fn link_mruby(workdir: &Path) {
-    let mruby_config = workdir.join("mruby").join("bin").join("mruby-config");
+    let mrb_cfg_bin = workdir
+        // On Windows, you don't need to manually change path separators "/" to "\\",
+        // because Rust std handles them automatically.
+        //
+        // Note: Modern Unix-like are compatible with the POSIX path separator `/`.
+        .join("mruby/bin/mruby-config");
+
+    #[allow(unreachable_patterns)]
+    let mruby_config = match mrb_cfg_bin {
+        #[cfg(windows)]
+        p => Some(p.with_extension("bat")).filter(|x| x.exists()),
+        #[cfg(not(windows))]
+        p if p.exists() => Some(p),
+        _ => None,
+    }
+    .expect(r#"The `mruby-config` executable file does not exist!"#);
```

Comment:
Could we write this as follows?

```suggestion
    let mruby_config = if cfg!(windows) {  mrb_cfg_bin.with_extension("bat") } else { mrb_cfg_bin }
    if !mruby_config.exists() {
      panic!(r#"The `mruby-config` executable file does not exist!"#);
    }
```